modules/openapi-generator/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -429,236 +431,185 @@ namespace {{packageName}}.Client
             return transformed;
         }
 
+        private ApiResponse<T> ExecClient<T>(Func<RestClient, RestResponse<T>> getResponse, Action<RestClientOptions> setOptions, RestRequest request, RequestOptions options, IReadableConfiguration configuration)
```

Comment:
can we add a dostring/comment explaining what this function does?

also we can we 4-space instead of tabs for indention?

Reply:
All done!